tests/test_kart.py

```diff
@@ -0,0 +1,164 @@
+import re
+import subprocess
+
+import pytest
+
+from sno.sqlalchemy.create_engine import gpkg_engine
+
+
+H = pytest.helpers.helpers()
+
+GPKG_IMPORTS = (
+    "archive,gpkg,table_ref",
+    [
+        pytest.param(
+            "gpkg-points", "nz-pa-points-topo-150k.gpkg", "POINTS", id="points"
+        ),
+        pytest.param(
+            "gpkg-polygons", "nz-waca-adjustments.gpkg", "POLYGONS", id="polygons"
+        ),
+    ],
+)
+
+
+@pytest.mark.slow
+@pytest.mark.e2e
+@pytest.mark.parametrize(*GPKG_IMPORTS)
+def test_kart(
+    archive,
+    gpkg,
+    table_ref,
+    data_archive,
+    tmp_path,
+    chdir,
+    cli_runner,
+    insert,
+    request,
+    monkeypatch,
+):
+    # Like test_e2e.py, but, uses a Kart branded repo instead of a Sno one.
```

Comment:
maybe swap this to be `test_e2e`, and make the Sno-flavoured one `test_e2e_sno`?

Reply:
Yeah I still need to do this - 
- when we turn on kart branding for new repos (not turned on as of this PR), test_e2e.py will become basically the same as test_kart - so at that point, I'll need to modify them as you suggest so that we have the standard e2e (kart) plus another e2e_sno.
- a lot of other tests will break (eg a test that creates a new repo with a new GPKG, and then tries to read the tracking table at sno_track). I'll need to update those to not break.
- eventually, I should upgrade the test .tgz repos that we have to be kart branded, not sno branded. This is lower priority though - it doesn't affect the user experience